index.js

```diff
@@ -1,5 +1,13 @@
 import React, { useState, useEffect, useRef } from "react";
-import { StyleSheet, View, Text, TouchableOpacity, I18nManager, Animated, Easing } from "react-native";
+import {
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
```

Comment:
make it editable.
 `TouchableOpacity` can be choosed to other using a new `TouchableComponent` prop.